app/routes/MyModal.tsx

```diff
@@ -0,0 +1,101 @@
+import { Dialog, Transition } from '@headlessui/react'
+import { Fragment, useState } from 'react'
+
+export default function MyModal() {
+  let [isOpen, setIsOpen] = useState(true)
```

Comment:
I saw that the Headless UI docs use `let` here, but it's best to use `const` so they don't get overwritten.